content/200-concepts/100-components/03-prisma-migrate/080-migrate-development-production.mdx

```diff
@@ -32,6 +32,7 @@ This command:
 1. Reruns the existing migration history in the [shadow database](shadow-database) in order to detect schema drift (edited or deleted migration file, or a manual changes to the database schema)
 1. Applies pending migrations to the shadow database (for example, new migrations created by colleagues)
 1. Generates a new migration from any changes you made to the Prisma schema before running `migrate dev`
+1. If it detects changes to the Prisma schema, it generates a new migration from these changes
```

Comment:
Isn't the point you added equivalent to the previous point? The way I read them, they express exactly the same (i.e. "a new migration is generated if there were changes to the Prisma schema").

So, with your current suggestions, I think 3. and 4. would be redundant.

However, I do find your version a bit clearer so my suggestion would be to drop the existing line and keep yours.

```suggestion
1. If it detects changes to the Prisma schema, it generates a new migration from these changes
```

<img width="527" alt="image" src="https://github.com/prisma/docs/assets/4058327/a781a72d-4d24-47a0-99d3-44f48c8b525e">